### DISCUSSION TITLE
V11 How to remove DataGridCell focus adorner

### DISCUSSION THREAD

**Discussion by Sciolo** [AvaloniaUI/Avalonia#11832] [Q&A] [answered]
Does anyone have an example of styling what I assume to be the focus adorner for a DataGridCell? I'm trying to remove or make invisible the 2 vertical black lines that appear at the left and right edges of data grid cells that have the focus.

Thanks,
James

**Comment by timunie** [comment]
best to look up original styles. Which Avalonia version do you use?

**Reply by Sciolo** [reply]
I'm using Avalonia v11 RC1. Maybe I'm being dense, but I couldn't find the control template for the DataGrid.

**Reply by timunie** [reply, marked as answer]
here you go: https://github.com/AvaloniaUI/Avalonia/blob/4eb75360da4453f3ca9221feafdd4c3be54e8667/src/Avalonia.Controls.DataGrid/Themes/Fluent.xaml#L146-L148

DataGrid has it's own package, so the style lives in there. ;-)

**Reply by Sciolo** [reply]
Thank you, I really appreciate your help.
